Library/Formula/scirius.rb

```diff
@@ -0,0 +1,49 @@
+class Scirius < Formula
+  desc "Rules manager for Suricata IDPS"
+  homepage "https://github.com/StamusNetworks/scirius"
+  url "https://github.com/StamusNetworks/scirius"
```

Comment:
Homebrew install things from source. Use `https://github.com/StamusNetworks/scirius/archive/scirius-1.0.tar.gz` as the URL, then run `brew fetch scirius`. It’ll give you a SHA256 checksum which you’ll then paste in the quotes after `sha256`.